tests/test_plotImageSubtractionCutouts.py

```diff
@@ -241,6 +242,23 @@ def test_generate_image_multisize_cutouts_with_metadata(self):
             # size (in inches) and the dpi (default=100), plus borders.
             self.assertEqual((im.height, im.width), (576, 645))
 
+    def test_generate_image_and_save_as_numpy(self):
+        """A different cutout size: the resulting cutout image is the same
+        size but shows more pixels.
+        """
+        config = plotImageSubtractionCutouts.PlotImageSubtractionCutoutsTask.ConfigClass()
+        config.sizes = [100]
+        with tempfile.TemporaryDirectory() as path:
+            cutouts = plotImageSubtractionCutouts.PlotImageSubtractionCutoutsTask(config=config,
+                                                                                  output_path=path)
+            cutouts.generate_image(self.science, self.template, self.difference, skyCenter, self.scale,
+                                   dia_source_id=506428274000265570, save_as_numpy=True)
+            numpy_dir_path = os.path.join(path, "raw_npy")
+            for file in os.listdir(numpy_dir_path):
+                image_with_channel = np.load(numpy_dir_path + "/" + file)
+                image = np.squeeze(image_with_channel, axis=0)
+                self.assertEqual((image.shape[0], image.shape[1]), (100, 100))
```

Comment:
I'd also compare the values of the image you read from the numpy file against the cutout you generated.

Reply:
This will be tricky because the ``generate_image`` function returns a plotted image (with all the metadata) and not just the raw cutout (unless I'm mistaken). The raw numpy cutouts, which are written to the file, are created and destroyed within that function and they are not returned against which I could compare after reading the files.

Reply:
Ah, of course.  Understood.